### PR TITLE
hard coded callbacks to avoid XSS. Closes #142

### LINKED ISSUE #142: XSS Vulnerability video-js.swf
We detected an XSS vulnerability:

Please try this link using safari or firefox, be sure to replace http://www.domain.com/path/to/ with a running webserver: 
http://www.domain.com/path/to/video-js.swf?readyFunction=alert

### PR DESCRIPTION
**Pull request by mmcc**

### CODE REVIEW

File: src/VideoJS.as [hunk 1]
```diff
@@ -94,39 +94,43 @@ package{
...
-            
+
             if(loaderInfo.parameters.mode != undefined){
                 _app.model.mode = loaderInfo.parameters.mode;
             }
-            
-            if(loaderInfo.parameters.eventProxyFunction != undefined){
+
+            // Hard coding these in for now until we can come up with a better solution for 5.0 to avoid XSS.
+            _app.model.jsEventProxyName = 'videojs.Flash.onEvent';
```
Comment (gkatsev) [inline src/VideoJS.as]: the code that uses this doesn't need to be changed?

File: src/VideoJS.as [hunk 2]
```diff
@@ -143,19 +147,22 @@ package{
...
-                try{
-                    ExternalInterface.call(_app.model.cleanEIString(loaderInfo.parameters.readyFunction), ExternalInterface.objectID);
-                }
-                catch(e:Error){
-                    if (loaderInfo.parameters.debug != undefined && loaderInfo.parameters.debug == "true") {
-                        throw new Error(e.message);
-                    }
+
+            // Hard coding this in for now until we can come up with a better solution for 5.0 to avoid XSS.
+            ExternalInterface.call('videojs.Flash.onReady', ExternalInterface.objectID);
```
Comment (gkatsev) [inline src/VideoJS.as]: For others, this is the main change.

**Comment by gkatsev** [comment]
Other than question on usage, LGTM.

File: src/VideoJS.as (hunk 1 above)
Comment (heff) [inline src/VideoJS.as]: We need to remove these settings from core.
https://github.com/videojs/video.js/blob/master/src/js/media/flash.js#L39

Comment (gkatsev) [inline src/VideoJS.as]: I also meant whether any changes in the swf are needed where these are used. It sounds like it doesn't.

**Pull request merged**